### ISSUE TITLE
[question] config.json

### ISSUE THREAD

**Issue by lutz** [ChordPro/chordpro#56]
If you have a sample configuration created in the program (GUI/CMD), does it contain all the settings that are currently working?

**Comment by sciurius** [comment]
Yes, it contains all the currently implemented settings, with their default
values.
